azuread/resource_domain_service_test.go

```diff
@@ -0,0 +1,207 @@
+package azuread
+
+import (
+	"fmt"
+	"net/http"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+	"github.com/terraform-providers/terraform-provider-azuread/azuread/helpers/ar"
+	"github.com/terraform-providers/terraform-provider-azuread/azuread/helpers/tf"
+)
+
+func TestAccAzureADDomainService_complete(t *testing.T) {
```

Comment:
Could we have a basic test that _only_ sets the required properties, and then an update test that goes from the basic config -> complete -> basic again.